client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -106,7 +106,7 @@ const LoggedInForm = React.createClass( {
 	componentWillMount() {
 		const { autoAuthorize, queryObject } = this.props.jetpackConnectAuthorize;
 		debug( 'Checking for auto-auth on mount', autoAuthorize );
-		if ( autoAuthorize ) {
+		if ( autoAuthorize || this.isCalypsoStartedConnection() ) {
```

Comment:
@roccotripaldi  I need your advice here... I'm not sure what that 'autoAuthorize' is ... is it coming from the API, isn't it? In which cases should come as true? I added the new code besides it, but I'm not sure if we should remove it